posthog/hogql/test/test_printer.py

```diff
@@ -520,6 +521,25 @@ def test_property_groups(self):
                 "nullIf(nullIf(events.mat_foo, ''), 'null')",
             )
 
+    def test_property_groups_person_properties(self):
+        if not settings.USE_PERSON_PROPERTIES_MAP_CUSTOM:
+            pytest.xfail("person_properties_map_custom not enabled")
```

Comment:
Can't use `override_settings` as the initial setting check in `posthog/clickhouse/property_groups.py` is done at module initialize time - this _should be_ short-lived so don't think it's worth going through the effort of refactoring it to facilitate easier testing

Reply:
This comment is great, but it should be in code.

Reply:
Makes sense, inlined the important part